### PR TITLE
Add slug link to email share and improves the mail's message text

### PR DESCRIPTION
**Pull request by clarissalimab**
This is how a share event looks with the placeholder text I added to the mailto link:

image

I'll just drop https://github.com/wwahammy/houdini/pull/21/commits/0065831b9e642eab2236fa388e803a1d8ce1496e if we decide on the mailto link.

### CODE REVIEW

File: app/views/common/_event_social_buttons.html.erb [hunk 1]
```diff
@@ -19,8 +19,7 @@
 	</div>
 
 	<div>
-		<a class='button--circle--large grey'>
-			<!--= on 'click' (open_modal_if_confirmed 'shareEmail') -->
+		<a class='button--circle--large grey' href='mailto:?subject=Check the <%= @event.name %> event from <%= @nonprofit.name %>!&amp;body=Hi,%0D%0A%0D%0A<%= @event.summary.present? ? @event.summary.concat("%0D%0A%0D%0A") : ''%>Follow this link for the event and participate: <%= url %>'>
```
Comment (wwahammy) [inline app/views/common/_event_social_buttons.html.erb]: Oh my goodness, no, that's a recipe for disaster. Use [mail_to](https://api.rubyonrails.org/v4.2/classes/ActionView/Helpers/UrlHelper.html#method-i-mail_to)

File: app/views/common/_event_social_buttons.html.erb [hunk 2]
```diff
@@ -19,10 +19,13 @@
 	</div>
 
 	<div>
-		<a class='button--circle--large grey'>
-			<!--= on 'click' (open_modal_if_confirmed 'shareEmail') -->
+		<%= mail_to('',
+			subject: "Check the #{@event.name} event from #{@nonprofit.name}!",
+			body: "Hi,\n\n#{@event.summary.present? ? [@event.summary, "\n\n"].join('') : ''}Follow this link for the event and participate: #{url}",
```
Comment (wwahammy) [inline app/views/common/_event_social_buttons.html.erb]: Does this work the same? I think it's clearer

```suggestion
			body: "Hi,\n\n#{@event.summary.present? ? "#{@event.summary}\n\n" : ''}Follow this link for the event and participate: #{url}",
```

File: app/views/common/_email_share_modal.html.erb [hunk 3]
```diff
@@ -20,10 +20,11 @@
 
 			<div class='field'>
 				<label if-branded='color, grey'>Your Message</label>
-				<textarea type='text' name='email[message]' style="height:100px;">Join me and donate to <%= fundraiser %>. <%= fundraiser_url %></textarea>
+				<textarea type='text' name='email[message]' style="height:100px;">Join me and donate to <%= fundraiser %>.<%= "\n\n" %>Follow this link to check it out: <%= fundraiser_url %></textarea>
 			</div>
 
 			<input type='hidden' name='email[from_email]' value='<%= current_user && current_user.email %>'>
+			<input type='hidden' name='email[from_name]' value='<%= current_user && current_user.name %>'>
```
Comment (wwahammy) [inline app/views/common/_email_share_modal.html.erb]: Are all these other changes still needed?

File: app/views/generic_mailer/generic_mail.text.erb [hunk 4]
```diff
@@ -1,4 +1,2 @@
 <%- # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later -%>
-<%= @from_name %> (<%= @from_email %>) has sent you a message:
```
Comment (wwahammy) [inline app/views/generic_mailer/generic_mail.text.erb]: Isn't this still needed?

**Review by wwahammy — CHANGES_REQUESTED** [review]
I think this is close to being complete. I had questions about a few things. It's kind of unclear to me if we should still be making changes to the email_share_modal here anymore. If we should, that's fine but I don't know.

File: app/views/common/_email_share_modal.html.erb (hunk 3 above)
Comment (clarissalimab) [reply app/views/common/_email_share_modal.html.erb]: No, I was waiting to drop this commit once Wendy approved the change.

File: app/views/generic_mailer/generic_mail.text.erb (hunk 4 above)
Comment (clarissalimab) [reply app/views/generic_mailer/generic_mail.text.erb]: Same as above!

File: app/views/common/_event_social_buttons.html.erb (hunk 2 above)
Comment (clarissalimab) [reply app/views/common/_event_social_buttons.html.erb]: It does! Nice catch, thanks

**Review by wwahammy — APPROVED** [review]
:rocket:

**Pull request merged**